src/app/shared/documentation-items/documentation-items.ts

```diff
@@ -601,12 +570,20 @@ const DOCS: {[key: string]: DocCategory[]} = {
 for (const category of DOCS[COMPONENTS]) {
   for (const doc of category.items) {
     doc.packageName = 'material';
+    doc.examples =
+      exampleNames
+        .filter(key => key.match(/^${doc.exampleSpecs.prefix}/) &&
```

Comment:
I don't think this syntax works in regexes, you'll need to do ``RegExp(`^${...}`)``